README.md

```diff
@@ -55,6 +55,28 @@
 
    Support to be included in [OpenBMC](https://github.com/openbmc). It is in planning, see [SPDM Integration](https://www.youtube.com/watch?v=PmgXkLJYI-E).
 
+7) Supported Architecture and Compiler

```

Comment:
Need to specify that these tables are for cross-compilation on x86 host.

Reply:
Yes, please let me know if the update is OK.

Or, please suggest the text directly.